schemas/skills/samples/complex-skillmanifest-2.1.preview-0.json

```diff
@@ -0,0 +1,184 @@
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-en.lu",
+                    "description": "English language model for the skill"
+                },
+                {
+                    "id": "SkillQnAModel-en",
+                    "name": "SkillBot QnA LU (English)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-QnA-en.lu",
+                    "description": "English language model for the skill (QnAMaker"
+                }
+            ],
+            "es-ES": [
+                {
+                    "id": "SkillLuModel-es-ES",
+                    "name": "SkillBot LU (Spanish-Spain)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-es-ES.lu",
+                    "description": "Spanish (Spain) language model for the skill"
+                }
+            ],
+            "es-MX": [
+                {
+                    "id": "SkillLuModel-es-MX",
+                    "name": "SkillBot LU (Spanish-Mexico)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-es-MX.lu",
+                    "description": "Spanish (Mexico) language model for the skill"
+                }
+            ]
+        },
+        "intents": {
+            "bookFlight": "#/activities/bookFlight",
+            "getWeather": "#/activities/getWeather",
+            "*": "#/activities/message"
+        }
+    },
+    "activities": {
+        "bookFlight": {
+            "description": "Books a flight",
+            "type": "event",
+            "name": "BookFlight",
+            "value": {
+                "$ref": "#/definitions/bookingInfo"
+            },
+            "resultValue": {
+                "$ref": "#/definitions/bookingInfo"
+            }
+        },
+        "getWeather": {
+            "description": "Retrieves and returns the weather for the user's location",
+            "type": "invoke",
+            "name": "GetWeather",
+            "value": {
+                "$ref": "#/definitions/location"
+            },
+            "resultValue": {
+                "$ref": "#/definitions/weatherReport"
+            }
+        },
+        "message": {
+            "type": "message",
+            "description": "Receives the user's' utterance and attempts to resolve it using the skill's LU models"
+        }
+    },
+    "definitions": {
+        "localeValue": {
+            "type": "object",
+            "properties": {
+                "locale": {
+                    "type": "string",
+                    "description": "The current user's locale ISO code"
+                }
+            }
+        },
+        "bookingInfo": {
+            "type": "object",
+            "required": [
+                "origin"
+            ],
+            "properties": {
+                "origin": {
```

Comment:
what types/primitivies are supported?

Reply:
The ones defined in [json-schema.org draft 07](https://json-schema.org/understanding-json-schema/reference/index.html)